src/sentry/static/sentry/app/components/organizations/globalSelectionHeader/globalSelectionHeader.jsx

```diff
@@ -98,6 +99,20 @@ class GlobalSelectionHeader extends React.Component {
       return;
     }
 
+    const hasSentry10 = new Set(this.props.organization.features).has('sentry10');
```

Comment:
We should talk about this in TCS as well @billyvg, we inconsistetly at times use `Set().has(...)` for features, other times just do `org.features.includes(...)`.

Reply:
yeah i noticed this and was confused.  seems like the second is simpler?